service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/judicial/NewHearingJudgeEventHandlerTest.java

```diff
@@ -189,4 +194,64 @@ void shouldHandleEditedHearingRolesWhenJudgeChanges() {
         verifyNoMoreInteractions(judicialService);
     }
 
+    @Test
+    void shouldCreateHearingRoleInFutureIfMultipleHearings() {
+        LocalDateTime future = LocalDateTime.now();
```

Comment:
"future" seems like a bad name for this variable. In the other test you call it "now" which makes more sense.

Reply:
agreed! changed back 😃